bin/build_extra_package.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+require 'manageiq/rpm_build/extra_package'
+
+Dir.chdir(ManageIQ::RPMBuild::ExtraPackage::PACKAGES_DIR) do
+  directories = (Dir["*"] - ["."]).reject { |o| !File.directory?(o) }
```

Comment:
Minor, but for consistency, this should probably be sorted

Reply:
Done